canonical-web-and-design/react-components
# Update to Vanilla 3.2.0 and prepare 0.34.2

## Done

- Updated Vanilla to 3.2
- Bump version of react components and rebuild storybook docs

## QA

- review package.json changes (make sure versions are bumped as expected)
  - all other code is autogenerated and can be ignored
- sanity check of the demo: https://react-components-758.demos.haus/
